skeleton/ci/source-repo/jenkins/Jenkinsfile

```diff
@@ -10,6 +10,9 @@ pipeline {
         ROX_CENTRAL_ENDPOINT = credentials('ROX_CENTRAL_ENDPOINT')
         GITOPS_AUTH_PASSWORD = credentials('GITOPS_AUTH_PASSWORD')
         QUAY_IO_CREDS = credentials('QUAY_IO_CREDS')
+        COSIGN_SECRET_PASSWORD = credentials('COSIGN_SECRET_PASSWORD')
+        COSIGN_SECRET_KEY = credentials('COSIGN_SECRET_KEY')
+        COSIGN_PUBLIC_KEY = credentials('COSIGN_PUBLIC_KEY')
```

Comment:
Might want to place these within the `sign-attest` stage only

Reply:
Yeah that's reasonable.

Reply:
Done in latest revision.